### PR TITLE
CMW handling in the verification pipeline

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix https://github.com/CCC-Attestation/attested-tls-poc/issues/37

### CODE REVIEW

**Review by paulhowardarm — APPROVED** [review]
LGTM - this matches the kind of change I would expect to see, given what we discussed. I'll leave any Golang-specific observations to other reviewers!

File: verification/api/handler.go [hunk 1]
```diff
@@ -320,9 +335,35 @@ func (o *Handler) SubmitEvidence(c *gin.Context) {
...
+		return
+	}
+
 	// read content-type and check against supported attestation formats
 	mediaType := c.Request.Header.Get("Content-Type")
 
+	if isCMW(mediaType) {
+		var w cmw.CMW
+
+		if err := w.Deserialize(evidence); err != nil {
```
Comment (yogeshbdeshpande) [inline verification/api/handler.go:355]: cmw Deserialize method needs a pointer to CMW object to set the fields.

Comment (yogeshbdeshpande) [reply verification/api/handler.go:355]: I meant : func (o *CMW) Deserialize(b []byte) error

Comment (thomas-fossati) [reply verification/api/handler.go:355]: as discussed in person, the code is ok as-is.

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**